src/machine.rs

```diff
@@ -1093,6 +1093,30 @@ impl OlmMachine {
             stream.for_each(move |item| send_user_identities_to_callback(callback_ref, item)).await;
         });
     }
+    /// Register a callback which will be called whenever a secret is received.
+    ///
+    /// `callback` should be a function that takes 2 arguments the secret name
+    /// and value (base64) and returns a Promise.
+    #[wasm_bindgen(js_name = "registerReceiveSecretCallback")]
+    pub async fn register_secret_receive_callback(&self, callback: Function) {
```

Comment:
please can you make these names match? Is it a "receive secret" callback or a "secret receive" callback?